src/app/components/CheckoutDialog.tsx

```diff
@@ -60,13 +60,14 @@ function CheckoutDialog(props: any) {
     });
     console.log(ticketsToDownload);
     finishTransaction();
+    /* TODO: Print tickets
     var dataStr =
       "data:text/json;charset=utf-8," +
       encodeURIComponent(JSON.stringify(ticketsToDownload));
     var dlAnchorElem: any = document.getElementById("downloadAnchorElem");
     dlAnchorElem.setAttribute("href", dataStr);
     dlAnchorElem.setAttribute("download", "scene.json");
-    dlAnchorElem.click();
+    dlAnchorElem.click(); */
```

Comment:
Warum drin lassen wenns nicht gebraucht wird?

Reply:
wird ab morgen gebraucht wenn ich die tickets downloadbar machen will wollte den code nicht verlieren

Reply:
Dann kommt das in das dementsprechende Ticket und nicht irgendwo random rein, das hat hier nichts zu suchen